src/mon/Monitor.cc

```diff
@@ -1054,7 +1054,11 @@ void Monitor::sync_obtain_latest_monmap(bufferlist &bl)
 
   dout(1) << __func__ << " obtained monmap e" << latest_monmap.epoch << dendl;
 
-  latest_monmap.encode(bl, quorum_features);
+  if (quorum_features) {
+    latest_monmap.encode(bl, quorum_features);
+  } else {
+    latest_monmap.encode(bl, CEPH_FEATURES_ALL);
+  }
```

Comment:
Oh，i suppose it will not be 0 when it is not the first time to do sync after osd started, use CEPH_FEATURES_ALL is ok if only used locally.